### ISSUE TITLE
extracting feature maps and logits

### ISSUE THREAD

**Issue by ghost** [HanyangGraduationProject-VSS/S3D-based-VSS#1]
S3Dmodel.predict() -> S3Dmodel.extract_logits_and_feature()
